src/main/java/com/minecolonies/coremod/network/messages/SaveScanMessage.java

```diff
@@ -41,7 +48,34 @@ public SaveScanMessage(final NBTTagCompound nbttagcompound, final String storeAt
     @Override
     public void fromBytes(@NotNull final ByteBuf buf)
     {
-        nbttagcompound = ByteBufUtils.readTag(buf);
+        final long NBTSizeTrackerMax = 2_000_971_52L;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)